osu.Framework.Tests/Visual/Sprites/TestSceneAnimation.cs

```diff
@@ -228,11 +231,26 @@ public void TestGotoZeroOnFirstFrameVisible()
             AddAssert("animation restarted from 0", () => animation.PlaybackPosition < 1000);
         }
 
+        [TestCase(0)]
+        [TestCase(48)]
+        public void TestGotoFrameBeforeLoaded(int frame)
+        {
+            AddStep("create new animation", () => animation = new TestAnimation(true, fontStore)
```

Comment:
Too late here but probably worth testing `startAtCurrentTime = false`, which is same behaviour actually.
```suggestion
        [Test]
        public void TestGotoFrameBeforeLoaded([Values(0, 48)] int frame, [Values(true, false)] bool startAtCurrentTime)
        {
            AddStep("create new animation", () => animation = new TestAnimation(startAtCurrentTime, fontStore)
```

Reply:
It doesn't really impact the fix in any capacity. `startAtCurrentTime` just determines whether to add the time consumed in `loadComplete()` to the animation's manual clock, so I'd say this would be superfluous.

Reply:
It kinda does, it's a scenario that hasn't been tested yet as far as I know (check `hasSeeked` bool), and this fix is allowing the scenario to be tested easily (previously you'll need to async load then seek), so probably worth covering it.